chainlit_app/Dockerfile.chainlit

```diff
@@ -0,0 +1,26 @@
+# Base image
+FROM python:3.10-slim
+
+# Set environment variables
+ENV PYTHONUNBUFFERED=1 \
+    PIP_NO_CACHE_DIR=1 \
+    PYTHONDONTWRITEBYTECODE=1
+
+# Set working directory
+WORKDIR /chainlit-app
```

Comment:
Not that this is hyphen but the the directory is underscore

`chainlit-app` vs, `chainlit_app`. We probably should be consistent to avoid confusion.

Reply:
Done